.lintstagedrc.json

```
@@ -0,0 +1,4 @@
+{
+  "src/*.{js,ts,json}": ["npx prettier --write --ignore-unknown"],
+  "./*.json": ["npx prettier --write --ignore-unknown"]
```

```suggestion
  "*": ["npx prettier --write --ignore-unknown"]
```
I think it'd probably just be easier to run prettier against everything?